### PR TITLE
check users online from eliza side

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala [hunk 1]
```diff
@@ -36,6 +36,11 @@ class ElizaController @Inject() (
     implicit val publicIdConfig: PublicIdConfiguration,
     clock: Clock) extends ElizaServiceController with Logging {
 
+  def areUsersOnline(users: String) = Action { request =>
+    val usersIds = users.split(",").map(_.toInt).map(Id[User](_)).toSet
+    Ok(notificationRouter.areUsersOnline(usersIds).map { case (id, res) => s""""$id":$res""" }.toArray.mkString("{", ",", "}"))
```
Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala]: I think you can just do `Ok(Json.toJson(notificationRouter.areUsersOnline(usersIds)))`. You may need to import one of the implicit formatters from the `Id` companion object.

File: kifi-backend/modules/shoebox/app/views/admin/organizations.scala.html [hunk 2]
```diff
@@ -134,6 +134,6 @@
 
     .tt-suggestion p {
     margin: 0;
-    }
+    }socket
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/views/admin/organizations.scala.html]: intended?

File: kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala (hunk 1 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala]: Also, do you even plan on calling this with a very large set of users? More than, say, 500? You may run into URL length limits.

Comment (eishay) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala]: good catch, thanks!
I think of using it to check onlinliness of users in an org to see who i can talk with. For now, only in admin, don't see it coming to production code anytime soon if at all. Yes, if there will be more then 500 we'll have to fix it.

File: kifi-backend/modules/shoebox/app/views/admin/organizations.scala.html (hunk 2 above)
Comment (eishay) [inline kifi-backend/modules/shoebox/app/views/admin/organizations.scala.html]: no, i thought i dropped it, thanks!

**Pull request merged**